### PR TITLE
added tests for Back End Services

### PR DESCRIPTION
**Pull request by KotliarovaK**

### CODE REVIEW

**Review by Valerii-Kryvokhyzha — APPROVED** [review]

File: ui_tests_playwright/automation/test/ui/mainSite/services/backEndDevelopment/BackEndDevelopmentContent.test.ts [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+	await baseDriverSteps.createsNewBrowserAndGoToUrl(UrlProvider.urlBuilder(UrlPath.BackEndDevelopment));
+});
+
+test(
+	qase(
+		5527,
+		'Check the Info container from the "Back-End Development" page @Regression @BackEndDevelopment @TSWEB-1208'
+	),
+	async () => {
+		const info = driver.getByTestId(BackEndServices.Info);
```
Comment (VladFedash) [inline ui_tests_playwright/automation/test/ui/mainSite/services/backEndDevelopment/BackEndDevelopmentContent.test.ts:24]: add lb

File: ui_tests_playwright/automation/test/ui/mainSite/services/backEndDevelopment/BackEndDevelopmentContent.test.ts [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+);
+
+test(
+	qase(
+		5530,
+		'Check section titles, image and CTA in "Our Featured Back-End Case Study" container from the "Back-End Development" page @Regression @BackEndDevelopment @TSWEB-1208'
+	),
+	async () => {
+		const caseStudyContainer = driver.getByTestId(BackEndServices.CaseStudy);
+
```
Comment (VladFedash) [inline ui_tests_playwright/automation/test/ui/mainSite/services/backEndDevelopment/BackEndDevelopmentContent.test.ts:168]: lb

File: ui_tests_playwright/automation/test/ui/mainSite/services/backEndDevelopment/BackEndDevelopmentContent.test.ts [hunk 3]
```diff
@@ -0,0 +1,402 @@
...
+	}
+);
+
+test(
+	qase(
+		5535,
+		'Check section titles and award cards in "Why Techstack Stands Out" container from the "Back-End Development" page @Regression @BackEndDevelopment @TSWEB-1208'
+	),
+	async () => {
+		const whyTechstackContainer = driver.getByTestId(BackEndServices.WhyTechstack);
```
Comment (VladFedash) [inline ui_tests_playwright/automation/test/ui/mainSite/services/backEndDevelopment/BackEndDevelopmentContent.test.ts:221]: make it grouped: 
constants are together without breaks, and actions are apart

File: ui_tests_playwright/automation/test/ui/mainSite/services/backEndDevelopment/BackEndDevelopmentContent.test.ts [hunk 4]
```diff
@@ -0,0 +1,402 @@
...
+			'High-load handling',
+			'Improved performance',
+			'Cost efficiency',
+			'Customization and flexibility',
+			'Reliability and stability',
+			'Future-proofing',
+		];
+
+		await expect(allSectionTitles).toHaveText(testData);
+		BackEndServices;
```
Comment (VladFedash) [inline ui_tests_playwright/automation/test/ui/mainSite/services/backEndDevelopment/BackEndDevelopmentContent.test.ts:270]: What does this code line mean?

File: ui_tests_playwright/automation/test/ui/mainSite/services/backEndDevelopment/BackEndDevelopmentContent.test.ts [hunk 5]
```diff
@@ -0,0 +1,402 @@
...
+	async () => {
+		const howWeCanWorkTogetherContainer = driver.getByTestId(BackEndServices.HowWeCanWorkTogether);
+
+		await expect(howWeCanWorkTogetherContainer.getByTestId(Container.SectionNumber)).toHaveText(['01', '02', '03']);
+
+		const allSectionTitles = howWeCanWorkTogetherContainer.getByTestId(Container.SectionTitle);
+		const testData = ['Full-Cycle Back-End\nDevelopment', 'Back-End Audit', 'Back-End Engineers\nfor Your Product'];
+
+		await expect(allSectionTitles).toHaveText(testData);
+
```
Comment (VladFedash) [inline ui_tests_playwright/automation/test/ui/mainSite/services/backEndDevelopment/BackEndDevelopmentContent.test.ts:349]: lb

File: ui_tests_playwright/automation/test/ui/mainSite/services/backEndDevelopment/BackEndDevelopmentActions.test.ts [hunk 6]
```diff
@@ -0,0 +1,226 @@
...
+);
+
+test(
+	qase(
+		5521,
+		'Check redirect by links in "Back-End Development Services for Any Industry" container from the "Back-End Development" page @Regression @BackEndDevelopment @TSWEB-1208'
+	),
+	async () => {
+		const BackendDevelopmentServicesContainer = driver.getByTestId(BackEndServices.BackendDevelopmentServices);
+
```
[the remark this answers is not in the document]
Comment (KotliarovaK) [reply ui_tests_playwright/automation/test/ui/mainSite/services/backEndDevelopment/BackEndDevelopmentActions.test.ts:159]: it's formatting type

File: ui_tests_playwright/automation/test/ui/mainSite/services/backEndDevelopment/BackEndDevelopmentContent.test.ts (hunk 4 above)
Comment (KotliarovaK) [reply ui_tests_playwright/automation/test/ui/mainSite/services/backEndDevelopment/BackEndDevelopmentContent.test.ts:270]: separates parts for better code reading

**Review by VladFedash — APPROVED** [review]

**Pull request merged**